cafebabel/templates/articles/translations/create.html

```diff
@@ -5,16 +5,19 @@
   {{ extra_js(['js/articles-tags.js', 'js/textarea-change.js']) }}
 {% endblock %}
 
+{% block body_class %}create-article-page{% endblock %}
```

Comment:
A little confused by the `body_class` names:
`{% block body_class %}profile-page edit{% endblock %}`
`{% block body_class %}tag-page-edit{% endblock %}`
`{% block body_class %}create-article-page{% endblock %}`

It would be less error prone to be consistent here and keep the same name structure.

Reply:
gute Idee ! I replaced `profile-page edit` https://github.com/cafebabel/cafebabel.com/pull/305/commits/6bddf4af7958390f7ad8a9b05f49b3b7e37b42c5 but I don't see how to be more consistent with `tag-page-edit` and  `create-article-page`. They name what the mean, don't they?

Reply:
Ah sorry, no you are right, in my head we had `create-article-page` AND `article-page-edit` but no 😝 I changed to `article-page-edit` then https://github.com/cafebabel/cafebabel.com/pull/305/commits/362ef772671bed58bab3b4407f31c65ab4df080d